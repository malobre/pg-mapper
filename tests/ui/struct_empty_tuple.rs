#[derive(pg_mapper::TryFromRow)]
struct MyTupleStruct();

fn main() {}
