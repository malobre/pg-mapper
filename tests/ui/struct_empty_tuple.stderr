error: `TryFromRow`: cannot derive for empty tuple-struct
 --> tests/ui/struct_empty_tuple.rs:2:21
  |
2 | struct MyTupleStruct();
  |                     ^^
